### PR TITLE
workflows: add make-gen-delta to chk for changes from 'make gen'

### PR DESCRIPTION
**Pull request by jimlambrt**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -81,3 +79,14 @@ test-database-down:
...
+.PHONY: copywrite
+copywrite:
+	copywrite headers
+
+.PHONY: fmt
+fmt:
+	gofumpt -w $$(find . -name '*.go')
+
+.PHONY: gen
+gen: docs copywrite
```
Comment (moduli) [inline Makefile:92]: Do we want to add `fmt` here?

File: .github/workflows/make-gen-delta.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+name: "make-gen-delta"
+on:
+  - workflow_dispatch
+  - push
+  - pull_request
```
[the remark this answers is not in the document]
Comment (jimlambrt) [reply .github/workflows/make-gen-delta.yml:5]: unfortunately, we can't.  If we remove pull then external contributors get no actions run for their PRs.

File: Makefile (hunk 1 above)
Comment (jimlambrt) [reply Makefile]: ty!

Comment (moduli) [reply Makefile]: Looks like it's failing because it can't find `gofumpt`

Comment (jimlambrt) [reply Makefile]: looking

**Review by johanbrandhorst — APPROVED** [review]

**Review by moduli — APPROVED** [review]

**Pull request merged**